src/server.tsx

```diff
@@ -122,26 +118,15 @@ const getArticleEpicTests = async (mvtId: number): Promise<Test[]> => {
     const regular = await fetchConfiguredArticleEpicTestsCached();
     const hardCoded = await getAllHardcodedTests();
 
-    return [
-        epicSeparateArticleCountTestUkAus,
-        epicSeparateArticleCountTestEuRow,
```

Comment:
Currently we've just removed the tests from this list, we haven't gone and fully cleaned up the article count test (i.e deleting `epicArticleCountTest` and removing references to variants in the `ContributionsEpicComponent`. We could do that now, or in a follow up PR?

Reply:
I think it would be good to clean that up at the same time

Reply:
Okay - we've just gone and removed all the old code!

Reply:
thanks!